### ISSUE TITLE
edrixs.rixs_2v1c_fort and similar

### ISSUE THREAD

**Issue by mpmdean** [EDRIXS/edrixs#39]
edrixs.rixs_2v1c_fort seems to rely on having called an ed calculation before and reading in hopping.in files.

Is this behavior optimal?

Either way, I think the docustring should make this clearer.

**Comment by shenmidelin** [comment]
Yes, xas_2v1c_fort and rixs_2v1c_fort  need inputs files hopping.* and coulomb.* , which are generated by ed_2v1c_fort.  

I agree with you, we should make it explicitly in docstring.

**Comment by mpmdean** [comment]
This is going to be covered by refactor, so closing.
